test/extension/handlers/lifecycle/handlers.go

```diff
@@ -0,0 +1,180 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package lifecycle contains the handlers for the lifecycle hooks.
+package lifecycle
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
```

Comment:
This is what's triggering the API diff - I don't think it should be an issue though.

Reply:
If possible, let's use `"sigs.k8s.io/yaml"` instead. 

It uses `gopkg.in/yaml.v2` internally, but then we are consistent with other places where we parse YAML.